test/Spring-test.js

```diff
@@ -167,26 +258,64 @@ describe('Spring', () => {
       },
     });
     TestUtils.renderIntoDocument(<App />);
+
+    // Initial render
+    expect(count).toEqual([0]);
+    mockRaf.manySteps(8);
+    expect(count).toEqual([
+      0,
```

Comment:
I don't have a strong opinion on this matter. Personally prefer to put values in each test. DRY is not for tests. Maybe you can reduce steps to 4, for example, so you will have less numbers to repeat. 

Oh and another concern - precision. It could vary. Maybe you should round values to, to say, 4-5 digits?